docs/usage/python-api.rst

```diff
@@ -570,6 +591,25 @@ Currently the following parameters are supported (more will be added if useful):
     df = result.df
     print(df.head())
 
+Instead of a latlon you may alternatively use an existing station id for which to summarize values in a manner of
+getting a more complete dataset:
+
+.. ipython:: python
+    :okwarning:
+
+    from wetterdienst.provider.dwd.observation import DwdObservationRequest
+    from wetterdienst import Parameter, Resolution
+
+    stations = DwdObservationRequest(
+        parameter=Parameter.TEMPERATURE_AIR_MEAN_200,
+        resolution=Resolution.HOURLY,
+        start_date=datetime(2022, 1, 1),
+        end_date=datetime(2022, 1, 20),
+    )
+
+    result = stations.summarize(latlon="02480")
```

Comment:
Should we rename the parameter `latlon` or split it into two different parameters something like`latlon` and `station_id`?

Reply:
Is there a nice way to do that? We would have to default both to `None` right?

```python
def summarize(latlon=None, station_id=None):
    if not (latlon or station_id) or (latlon and station_id):
         raise ValueError("use either latlon (float, float) or station_id (str) to get interpolated values")
```

Reply:
After doing a bit of research again, it seems there's no nice solution for that problem, so I decided to split up the method in two:
- `summarize` (as before)
- `summarize_by_station_id`

Do you think the original method should be renamed to something like `summarize_by_latlon`?

(same applies for `interpolate`)